src/components/RenderHTML/renderHTMLPropTypes.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
```

Comment:
I think we should make this file look like this:

```javascript
const propTypes = {
    /** HTML string to render */
    html: PropTypes.string.isRequired,

    /** Optional debug flag */
    debug: PropTypes.bool,
};

const defaultProps = {
    debug: false,
};

export {propTypes, defaultProps};
```

Then in `BaseRenderHTML`:

```javascript
import {
    propTypes as renderHTMLPropTypes,
    defaultProps as renderHTMLDefaultProps,
} from './renderHTMLPropTypes';

const propTypes = {
    // Whether text elements should be selectable
    textSelectable: PropTypes.bool,
    
    ...renderHTMLPropTypes,
};

const defaultProps = {
    textSelectable: false,
    ...renderHTMLDefaultProps,
};
...
...
BaseRenderHTML.propTypes = propTypes;
BaseRenderHTML.defaultProps = defaultProps;
BaseRenderHTML.displayName = 'BaseRenderHTML';

export default BaseRenderHTML;
```